pycaret/containers/models/time_series/time_series_forecasting.py

```diff
@@ -90,7 +90,7 @@ def get_container_default_engines() -> Dict[str, str]:
     return default_engines
 
 
-class TimeSeriesContainer(ModelContainer):
+class TimeSeriesForecastingContainer(ModelContainer):
```

Comment:
ok with the name change here. But I would preferr to move all time series containers under their own folder.

e.g. pycaret --> containers --> models --> time_series (new folder) --> `clustering.py` & `forecasting.py`